tests/utils/sc_linac/test_cavity.py

```diff
@@ -345,6 +345,11 @@ def test_edm_macro_string_rack_b(cavity):
     assert cav.edm_macro_string == "C=5,RFS=1B,R=B,CM=ACCL:L0B:01,ID=01,CH=1"
 
 
+def test_cryo_edm_macro_string(cavity):
+    cavity = MACHINE.cryomodules["01"].cavities[1]
```

Comment:
Ok this is my fault but the tests you modeled this after are incorrect; we shouldn't be creating a new cavity, it's generated by the fixture in line 49

Reply:
Ah, okay, gotcha!